core/DataTable/Row.php

```diff
@@ -475,7 +470,7 @@ private function isSummableColumn($columnName)
      */
     public function sumRow(Row $rowToSum, $enableCopyMetadata = true, $aggregationOperations = false)
     {
-        foreach ($rowToSum->getColumnsRaw() as $columnToSumName => $columnToSumValue) {
+        foreach ($rowToSum->getColumns() as $columnToSumName => $columnToSumValue) {
```

Comment:
This was just something that I noticed while working on it. `getColumnsRaw ` and `getColumns` does now do the same since we removed callable columns a few weeks ago. I can remove it from the PR if it annoys anyone